chatBotManager.js

```diff
@@ -1,5 +1,15 @@
 const DiscordBot = require('./discordBot')
 const { SlackBot } = require('./slackBot');
+const User = require('./user');
+const DiscordCommands = require('./discord-command');
```

Comment:
we already have bot class for each channel, you could look into `discordBot.js`, `slackBot.js`